python/cudf/cudf/core/_base_index.py

```diff
@@ -796,6 +796,12 @@ def fillna(self, value, downcast=None):
 
         return super().fillna(value=value)
 
+    def _get_level_names(self):
+        """
+        Return a name or list of names with None replaced by the level number.
+        """
+        return 0 if self.name is None else self.name
```

Comment:
This docstring is a "generic" one, but this function never returns a list.

Reply:
Dropped this method altogether.